pkg/pb/metainfo.proto

```diff
@@ -24,6 +31,72 @@ service Metainfo {
     rpc ProjectInfo(ProjectInfoRequest) returns (ProjectInfoResponse);
 }
 
+message BucketItem {
+    bytes             name = 1;
+    encryption.Cipher path_cipher = 2;
+    bytes             attribution_id = 3;
+
+    google.protobuf.Timestamp created_at = 4;
+
+    int64                           default_segment_size = 5;
+    pointerdb.RedundancyScheme      default_redundancy_scheme = 6;
+    encryption.EncryptionParameters default_encryption_parameters = 7;
+}
+
+message BucketListItem {
+    bytes             name = 1;
+    bytes             attribution_id = 2;
```

Comment:
why attribution_id in a bucket list? i don't think we need that in a list

Reply:
when we list buckets i thought we would want to see which ones are attribution buckets but i really don't know. I can go either way.  @egonelbre @mniewrzal thoughts?

Reply:
I guess `attribution_id` can be obtained with `GetBucket` if it will be necessary.